lib/main_containers.pm

```diff
@@ -153,7 +153,7 @@ sub load_host_tests_docker {
     unless (is_sle("<=15") && is_aarch64) {
         # these 2 packages are not avaiable for <=15 (aarch64 only)
         # zypper-docker is only available on SLES < 15-SP6
-        loadtest 'containers/zypper_docker' if (is_sle("<15-SP6"));
+        loadtest 'containers/zypper_docker' if (is_sle("<15-SP6") || is_leap("<15.6") || is_sle_micro || is_leap_micro);
```

Comment:
Older than slem 6.0, no?

Reply:
on SLEM actually not at all - see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18770/files#diff-5b76203f35ae2252c0aab9d1e126c9c90f60815c772cc07e098ff35b358f9bd9L155

Reply:
I think we were only missing Leap.

Reply:
true